tests/contrib/io/azure/test_csv_blob.py

```diff
@@ -62,6 +74,103 @@ def make_data_set(load_args=None, save_args=None):
     return make_data_set
 
 
+@pytest.fixture
+def versioned_blob_csv_data_set(load_version, save_version):
```

Comment:
Because `BlockBlobService` was initialised in the constructor, we want to mock it here (see how S3 object is mocked in other S3 dataset test).